### PR TITLE
add examples for lang redirections

### PR DESCRIPTION
**Pull request by timoleo23**
Hi @sebader , I added an example to create the string to redirect based on language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
...
 
 Use this command to deploy the ARM template - replace the **backends** parameter with your individual, semicolon-separated list of Event URLs, starting with the language code. Within each language code you can have multiple URLs. In this case, those will be load-balanced as well. There always needs to be at least one URL for English (`en`) as this is used as the fallback option. Also, update **locationSecondary** based on your preferences.
+
+Let's take the following example:
+- For **English (en)**, you need to redirect users to URL's **https://URL_EN_1** and **https://URL_EN_2**
+- For **French (fr)**, you need to redirect users to URL's **https://URL_FR_1**, **https://URL_FR_2** and **https://URL_FR_3**
+- For **German (de)**, you need to redirect userd to URL's **https://URL_GE_1**
+
+Here is how you will create the "backends" string for such a configuration:
+backends="en=https://URL_EN_1;https://URL_EN_2,fr=https://URL_FR_1;https://URL_FR_2;https://URL_FR_3,de=https://URL_GE_1"
```
Comment (sebader) [inline README.md]: can you put it in ticks so it gets rendered nicely?

`backends="en=https://URL_EN_1;https://URL_EN_2,fr=https://URL_FR_1;https://URL_FR_2;https://URL_FR_3,de=https://URL_GE_1"`

File: README.md [hunk 2]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
 **Note**: This mode currently only supports a list of URLs that is not longer than approx. 14,000 characters.
 
 Use this command to deploy the ARM template - replace the **backends** parameter with your individual, semicolon-separated list of Event URLs, starting with the language code. Within each language code you can have multiple URLs. In this case, those will be load-balanced as well. There always needs to be at least one URL for English (`en`) as this is used as the fallback option. Also, update **locationSecondary** based on your preferences.
+
+Let's take the following example:
+- For **English (en)**, you need to redirect users to URL's **https://URL_EN_1** and **https://URL_EN_2**
```
Comment (sebader) [inline README.md]: maybe put english last (after french and german) as it is also the fallback

File: README.md [hunk 3]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
...
+- For **French (fr)**, you need to redirect users to URL's **https://URL_FR_1**, **https://URL_FR_2** and **https://URL_FR_3**
```
Comment (sebader) [inline README.md]: maybe something like 

 For **French (fr)**, you have the following three URLs to to redirect and load balance users to:

File: README.md [hunk 4]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
...
+- For **German (de)**, you need to redirect userd to URL's **https://URL_GE_1**
```
Comment (sebader) [inline README.md]: typo userd

File: README.md [hunk 5]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
...
+
+Here is how you will create the "backends" string for such a configuration:
+backends="en=https://URL_EN_1;https://URL_EN_2,fr=https://URL_FR_1;https://URL_FR_2;https://URL_FR_3,de=https://URL_GE_1"
+
+Note: all other languages, not specified inthe backends definition, will fall into "en" as a default.
```
Comment (sebader) [inline README.md]: typo inthe

**Review by sebader — CHANGES_REQUESTED** [review]

**Comment by timoleo23** [comment]
Changes commited

File: README.md [hunk 6]
```diff
@@ -71,14 +71,14 @@ To use this more, your list of backends must be formatted different. See the fol
 Use this command to deploy the ARM template - replace the **backends** parameter with your individual, semicolon-separated list of Event URLs, starting with the language code. Within each language code you can have multiple URLs. In this case, those will be load-balanced as well. There always needs to be at least one URL for English (`en`) as this is used as the fallback option. Also, update **locationSecondary** based on your preferences.
 
 Let's take the following example:
-- For **English (en)**, you need to redirect users to URL's **https://URL_EN_1** and **https://URL_EN_2**
-- For **French (fr)**, you need to redirect users to URL's **https://URL_FR_1**, **https://URL_FR_2** and **https://URL_FR_3**
-- For **German (de)**, you need to redirect userd to URL's **https://URL_GE_1**
+- For **French (fr)**, you have the following three URLs to redirect and load balance users to: **https://URL_FR_1**, **https://URL_FR_2** and **https://URL_FR_3**
+- For **German (de)**, you have the following URL to redirect and load balance users to: **https://URL_GE_1**
```
Comment (sebader) [inline README.md]: well, technically now for German you don't have anything to load balance ;)
So maybe just say 

For **German (de)**, you have the following single URL to redirect users to: **https://URL_GE_1**

**Review by sebader — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -69,6 +69,17 @@ To use this more, your list of backends must be formatted different. See the fol
...
+- For **English (en)**, you have the following two URLs to redirect and load balance users to: **https://URL_EN_1** and **https://URL_EN_2**
+
+Here is how you will create the "backends" string for such a configuration:
+`backends="en=https://URL_EN_1;https://URL_EN_2,fr=https://URL_FR_1;https://URL_FR_2;https://URL_FR_3,de=https://URL_GE_1"`
+
+Note: all other languages, not specified in the backends definition, will fall into "en" as a default.
+
 ```
 az deployment group create -g  myresource-group --template-file .\main.json -p prefix=myprefix -p locationSecondary=westeurope -p loadBalancingMode=userLanguage -p backends="de=https://teams.microsoft.com/l/meetup-join/19%3ameeting_GERMAN1;fr=https://teams.microsoft.com/l/meetup-join/19%3ameeting_FRENCH1;en=https://teams.microsoft.com/l/meetup-join/19%3ameeting_ENGLISH1,https://teams.microsoft.com/l/meetup-join/19%3ameeting_ENGLISH2"
```
Comment (sebader) [inline README.md]: ah, and can you update this example to use your URLs from above? thanks!

**Review by sebader — CHANGES_REQUESTED** [review]

**Comment by timoleo23** [comment]
changes commited

**Review by sebader — APPROVED** [review]
beautiful! thanks so much for your contribution

**Pull request merged**